integration-tests/lib/zendeskParameters.ts

```diff
@@ -0,0 +1,40 @@
+const message = ' is not set. Please check your test environment file.'
+
+const getEndUsername = () => {
+  if (process.env.ZENDESK_END_USERNAME) {
+    return process.env.ZENDESK_END_USERNAME
```

Comment:
You can avoid repeating yourself in these methods by using the fact that you can do process.env['ENV_VARIABLE_NAME']. Then you can write your logic for checking if the variable is set once in one function, and use it in all the others, passing in a string representing the variable name (e.g. 'ZENDESK_END_USERNAME')

Reply:
@gdsandrewmccafferty thanks for that tip. Done.